doc/install.md

```diff
@@ -12,11 +12,13 @@ You can also build the executables from scratch.
 
 2. You can fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal.
 
-3. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
+3. Make sure you have the necessary libraries installed `apt-get update && apt-get -y install build-essential pkg-config autoconf nodejs gnutls-dev`
```

Comment:
Is the windows build stable though? Otherwise we should probably not include it for now.
I imagine it involves some obnoxious steps like using Mingw or Cygwin and static builds for ffmpeg.

I would happily try it out though since I have dual boot on this machine.

Reply:
> Is the windows build stable though?

Not sure, but I thought I'd note it.

We can include the OSX/Linux commands in this PR and come back and add Windows commands later on.